modules/player/src/main/java/unnamed/mmo/modifiers/Modifier.java

```diff
@@ -0,0 +1,10 @@
+package unnamed.mmo.modifiers;
+
+public interface Modifier<T> {
+
+    T modifierAmount();
```

Comment:
Generic is probably unnecessary here, I doubt there are any modifiers that cannot be modeled by a double (also currently hardcoded to double everywhere anyway)

Reply:
It is most likely unnecessary, but I would like to leave it in just in case we decide to expand upon this in the future.